src/constants/label.js

```diff
@@ -122,7 +122,7 @@ export function localizeLayers(layers, locales) {
 export const localizedName = [
   "let",
   "localizedName",
-  "",
+  ["get", "name"],
```

Comment:
Change this test to expect “Null Island” instead of the empty string:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/0eb949eaa1618b756ce80a04e8e5ac27c3122492/test/spec/label.js#L243-L252